<!DOCTYPE html>
<!-- START: inst/pkgdown/templates/layout.html --><!-- Generated by pkgdown: do not edit by hand --><html lang="en" data-bs-theme="auto"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta charset="utf-8"><title>Marine Biological Data Mobilization Workshop 2024: DNA-derived Data Extension</title><meta name="viewport" content="width=device-width, initial-scale=1"><script src="../assets/themetoggle.js"></script><link rel="stylesheet" type="text/css" href="../assets/styles.css"><script src="../assets/scripts.js" type="text/javascript"></script><!-- mathjax --><script type="text/x-mathjax-config">
    MathJax.Hub.Config({
      config: ["MMLorHTML.js"],
      jax: ["input/TeX","input/MathML","output/HTML-CSS","output/NativeMML", "output/PreviewHTML"],
      extensions: ["tex2jax.js","mml2jax.js","MathMenu.js","MathZoom.js", "fast-preview.js", "AssistiveMML.js", "a11y/accessibility-menu.js"],
      TeX: {
        extensions: ["AMSmath.js","AMSsymbols.js","noErrors.js","noUndefined.js"]
      },
      tex2jax: {
        inlineMath: [['\\(', '\\)']],
        displayMath: [ ['$$','$$'], ['\\[', '\\]'] ],
        processEscapes: true
      }
    });
    </script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><!-- Responsive Favicon for The Carpentries --><link rel="apple-touch-icon" sizes="180x180" href="../favicons/incubator/apple-touch-icon.png"><link rel="icon" type="image/png" sizes="32x32" href="../favicons/incubator/favicon-32x32.png"><link rel="icon" type="image/png" sizes="16x16" href="../favicons/incubator/favicon-16x16.png"><link rel="manifest" href="../favicons/incubator/site.webmanifest"><link rel="mask-icon" href="../favicons/incubator/safari-pinned-tab.svg" color="#5bbad5"><meta name="msapplication-TileColor" content="#da532c"><meta name="theme-color" media="(prefers-color-scheme: light)" content="white"><meta name="theme-color" media="(prefers-color-scheme: dark)" content="black"></head><body>
    <header id="top" class="navbar navbar-expand-md top-nav incubator"><svg xmlns="http://www.w3.org/2000/svg" class="d-none"><symbol id="check2" viewbox="0 0 16 16"><path d="M13.854 3.646a.5.5 0 0 1 0 .708l-7 7a.5.5 0 0 1-.708 0l-3.5-3.5a.5.5 0 1 1 .708-.708L6.5 10.293l6.646-6.647a.5.5 0 0 1 .708 0z"></path></symbol><symbol id="circle-half" viewbox="0 0 16 16"><path d="M8 15A7 7 0 1 0 8 1v14zm0 1A8 8 0 1 1 8 0a8 8 0 0 1 0 16z"></path></symbol><symbol id="moon-stars-fill" viewbox="0 0 16 16"><path d="M6 .278a.768.768 0 0 1 .08.858 7.208 7.208 0 0 0-.878 3.46c0 4.021 3.278 7.277 7.318 7.277.527 0 1.04-.055 1.533-.16a.787.787 0 0 1 .81.316.733.733 0 0 1-.031.893A8.349 8.349 0 0 1 8.344 16C3.734 16 0 12.286 0 7.71 0 4.266 2.114 1.312 5.124.06A.752.752 0 0 1 6 .278z"></path><path d="M10.794 3.148a.217.217 0 0 1 .412 0l.387 1.162c.173.518.579.924 1.097 1.097l1.162.387a.217.217 0 0 1 0 .412l-1.162.387a1.734 1.734 0 0 0-1.097 1.097l-.387 1.162a.217.217 0 0 1-.412 0l-.387-1.162A1.734 1.734 0 0 0 9.31 6.593l-1.162-.387a.217.217 0 0 1 0-.412l1.162-.387a1.734 1.734 0 0 0 1.097-1.097l.387-1.162zM13.863.099a.145.145 0 0 1 .274 0l.258.774c.115.346.386.617.732.732l.774.258a.145.145 0 0 1 0 .274l-.774.258a1.156 1.156 0 0 0-.732.732l-.258.774a.145.145 0 0 1-.274 0l-.258-.774a1.156 1.156 0 0 0-.732-.732l-.774-.258a.145.145 0 0 1 0-.274l.774-.258c.346-.115.617-.386.732-.732L13.863.1z"></path></symbol><symbol id="sun-fill" viewbox="0 0 16 16"><path d="M8 12a4 4 0 1 0 0-8 4 4 0 0 0 0 8zM8 0a.5.5 0 0 1 .5.5v2a.5.5 0 0 1-1 0v-2A.5.5 0 0 1 8 0zm0 13a.5.5 0 0 1 .5.5v2a.5.5 0 0 1-1 0v-2A.5.5 0 0 1 8 13zm8-5a.5.5 0 0 1-.5.5h-2a.5.5 0 0 1 0-1h2a.5.5 0 0 1 .5.5zM3 8a.5.5 0 0 1-.5.5h-2a.5.5 0 0 1 0-1h2A.5.5 0 0 1 3 8zm10.657-5.657a.5.5 0 0 1 0 .707l-1.414 1.415a.5.5 0 1 1-.707-.708l1.414-1.414a.5.5 0 0 1 .707 0zm-9.193 9.193a.5.5 0 0 1 0 .707L3.05 13.657a.5.5 0 0 1-.707-.707l1.414-1.414a.5.5 0 0 1 .707 0zm9.193 2.121a.5.5 0 0 1-.707 0l-1.414-1.414a.5.5 0 0 1 .707-.707l1.414 1.414a.5.5 0 0 1 0 .707zM4.464 4.465a.5.5 0 0 1-.707 0L2.343 3.05a.5.5 0 1 1 .707-.707l1.414 1.414a.5.5 0 0 1 0 .708z"></path></symbol></svg><a class="visually-hidden-focusable skip-link" href="#main-content">Skip to main content</a>
  <div class="container-fluid top-nav-container">
    <div class="col-md-8">
      <div class="large-logo">
        <img id="incubator-logo" alt="Marine data mobilization workshop for Biology and Ecosystem Essential Ocean Variables (Bio-Eco EOV) as a Contribution to the UN Decade on Ocean Science for Sustainable Development" src="../assets/images/incubator-logo.svg"><span class="badge text-bg-warning">
          <abbr title="This lesson is in the alpha phase, which means that it has been taught once and lesson authors are iterating on feedback.">
            <a href="https://cdh.carpentries.org/the-lesson-life-cycle.html#field-testing-alpha-stage" class="external-link alert-link">
              <i aria-hidden="true" class="icon" data-feather="alert-triangle" style="border-radius: 5px"></i>
              Alpha
            </a>
            <span class="visually-hidden">This lesson is in the alpha phase, which means that it has been taught once and lesson authors are iterating on feedback.</span>
          </abbr>
        </span>

      </div>
    </div>
    <div class="selector-container">
      <div id="theme-selector">
        <li class="nav-item dropdown" id="theme-button-list">
          <button class="btn btn-link nav-link px-0 px-lg-2 dropdown-toggle d-flex align-items-center" id="bd-theme" type="button" aria-expanded="false" data-bs-toggle="dropdown" data-bs-display="static" aria-label="Toggle theme (auto)">
            <svg class="bi my-1 theme-icon-active"><use href="#circle-half"></use></svg><i data-feather="chevron-down"></i>
          </button>
          <ul class="dropdown-menu dropdown-menu-end" aria-labelledby="bd-theme-text"><li>
              <button type="button" class="btn dropdown-item d-flex align-items-center" data-bs-theme-value="light" aria-pressed="false">
                <svg class="bi me-2 theme-icon"><use href="#sun-fill"></use></svg>
                Light
                <svg class="bi ms-auto d-none"><use href="#check2"></use></svg></button>
            </li>
            <li>
              <button type="button" class="btn dropdown-item d-flex align-items-center" data-bs-theme-value="dark" aria-pressed="false">
                <svg class="bi me-2 theme-icon"><use href="#moon-stars-fill"></use></svg>
                Dark
                <svg class="bi ms-auto d-none"><use href="#check2"></use></svg></button>
            </li>
            <li>
              <button type="button" class="btn dropdown-item d-flex align-items-center active" data-bs-theme-value="auto" aria-pressed="true">
                <svg class="bi me-2 theme-icon"><use href="#circle-half"></use></svg>
                Auto
                <svg class="bi ms-auto d-none"><use href="#check2"></use></svg></button>
            </li>
          </ul></li>
      </div>

      <div class="dropdown" id="instructor-dropdown">
        <button class="btn btn-secondary dropdown-toggle bordered-button" type="button" id="dropdownMenu1" data-bs-toggle="dropdown" aria-expanded="false">
          <i aria-hidden="true" class="icon" data-feather="eye"></i> Instructor View <i data-feather="chevron-down"></i>
        </button>
        <ul class="dropdown-menu" aria-labelledby="dropdownMenu1"><li><button class="dropdown-item" type="button" onclick="window.location.href='../edna-extension.html';">Learner View</button></li>
        </ul></div>
    </div>
  </div>
  <hr></header><nav class="navbar navbar-expand-xl bottom-nav incubator" aria-label="Main Navigation"><div class="container-fluid nav-container">
    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle Navigation">
      <span class="navbar-toggler-icon"></span>
      <span class="menu-title">Menu</span>
    </button>
    <div class="nav-logo">
      <img class="small-logo" alt="Marine data mobilization workshop for Biology and Ecosystem Essential Ocean Variables (Bio-Eco EOV) as a Contribution to the UN Decade on Ocean Science for Sustainable Development" src="../assets/images/incubator-logo-sm.svg"></div>
    <div class="lesson-title-md">
      Marine Biological Data Mobilization Workshop 2024
    </div>
    <div class="search-icon-sm">
      <!-- TODO: do not show until we have search
        <i role="img" aria-label="Search the All In One page" data-feather="search"></i>
      -->
    </div>
    <div class="desktop-nav">
      <ul class="navbar-nav me-auto mb-2 mb-lg-0"><li class="nav-item">
          <span class="lesson-title">
            Marine Biological Data Mobilization Workshop 2024
          </span>
        </li>
        <li class="nav-item">
          <a class="nav-link" href="../instructor/key-points.html">Key Points</a>
        </li>
        <li class="nav-item">
          <a class="nav-link" href="../instructor/instructor-notes.html">Instructor Notes</a>
        </li>
        <li class="nav-item">
          <a class="nav-link" href="../instructor/images.html">Extract All Images</a>
        </li>
        <li class="nav-item dropdown">
          <button class="nav-link dropdown-toggle" id="navbarDropdown" data-bs-toggle="dropdown" aria-expanded="false">
            More <i data-feather="chevron-down"></i>
          </button>
          <ul class="dropdown-menu" aria-labelledby="navbarDropdown"><hr><li><a class="dropdown-item" href="reference.html">Reference</a></li><li><a class="dropdown-item" href="about.html">About</a></li><li><a class="dropdown-item" href="helpful-tutorials.html">Helpful Tutorials</a></li><li><a class="dropdown-item" href="edna-extension.html">DNA-derived Data Extension</a></li><li><a class="dropdown-item" href="acoustic-telemetry.html">Acoustic (and Satellite) Telemetry</a></li>
          </ul></li>
      </ul></div>
    <!--
    <form class="d-flex col-md-2 search-form">
      <fieldset disabled>
      <input class="form-control me-2 searchbox" type="search" placeholder="" aria-label="">
        <button class="btn btn-outline-success tablet-search-button"  type="submit">
          <i class="search-icon" data-feather="search" role="img" aria-label="Search the All In One page"></i>
        </button>
      </fieldset>
    </form>
    -->
    <a id="search-button" class="btn btn-primary" href="../instructor/aio.html" role="button" aria-label="Search the All In One page">Search the All In One page</a>
  </div><!--/div.container-fluid -->
</nav><div class="col-md-12 mobile-title">
  Marine Biological Data Mobilization Workshop 2024
</div>

<aside class="col-md-12 lesson-progress"><div style="width: NA%" class="percentage">
    NA%
  </div>
  <div class="progress incubator">
    <div class="progress-bar incubator" role="progressbar" style="width: NA%" aria-valuenow="NA" aria-label="Lesson Progress" aria-valuemin="0" aria-valuemax="100">
    </div>
  </div>
</aside><div class="container">
      <div class="row">
        <!-- START: inst/pkgdown/templates/navbar.html -->
<div id="sidebar-col" class="col-lg-4">
  <div id="sidebar" class="sidebar">
      <nav aria-labelledby="flush-headingEleven"><button role="button" aria-label="close menu" alt="close menu" aria-expanded="true" aria-controls="sidebar" class="collapse-toggle" data-collapse="Collapse " data-episodes="Episodes ">
          <i class="search-icon" data-feather="x" role="img"></i>
        </button>
        <div class="sidebar-inner">
          <div class="row mobile-row" id="theme-row-mobile">
            <div class="col" id="theme-selector">
              <li class="nav-item dropdown" id="theme-button-list">
                <button class="btn btn-link nav-link px-0 px-lg-2 dropdown-toggle d-flex align-items-center" id="bd-theme" type="button" aria-expanded="false" data-bs-toggle="dropdown" data-bs-display="static" aria-label="Toggle theme (auto)">
                  <svg class="bi my-1 theme-icon-active"><use href="#circle-half"></use></svg><span class="d-lg-none ms-1" id="bd-theme-text">Toggle Theme</span>
                </button>
                <ul class="dropdown-menu dropdown-menu-right" aria-labelledby="bd-theme-text"><li>
                    <button type="button" class="btn dropdown-item d-flex align-items-center" data-bs-theme-value="light" aria-pressed="false">
                      <svg class="bi me-2 theme-icon"><use href="#sun-fill"></use></svg>
                      Light
                      <svg class="bi ms-auto d-none"><use href="#check2"></use></svg></button>
                  </li>
                  <li>
                    <button type="button" class="btn dropdown-item d-flex align-items-center" data-bs-theme-value="dark" aria-pressed="false">
                      <svg class="bi me-2 theme-icon"><use href="#moon-stars-fill"></use></svg>
                      Dark
                      <svg class="bi ms-auto d-none"><use href="#check2"></use></svg></button>
                  </li>
                  <li>
                    <button type="button" class="btn dropdown-item d-flex align-items-center active" data-bs-theme-value="auto" aria-pressed="true">
                      <svg class="bi me-2 theme-icon"><use href="#circle-half"></use></svg>
                      Auto
                      <svg class="bi ms-auto d-none"><use href="#check2"></use></svg></button>
                  </li>
                </ul></li>
            </div>
          </div>
          <div class="row mobile-row">
            <div class="col">
              <div class="sidenav-view-selector">
                <div class="accordion accordion-flush" id="accordionFlush9">
                  <div class="accordion-item">
                    <h2 class="accordion-header" id="flush-headingNine">
                      <button class="accordion-button collapsed" id="instructor" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapseNine" aria-expanded="false" aria-controls="flush-collapseNine">
                        <i id="eye" aria-hidden="true" class="icon" data-feather="eye"></i> Instructor View
                      </button>
                    </h2>
                    <div id="flush-collapseNine" class="accordion-collapse collapse" aria-labelledby="flush-headingNine" data-bs-parent="#accordionFlush2">
                      <div class="accordion-body">
                        <a href="../edna-extension.html">Learner View</a>
                      </div>
                    </div>
                  </div><!--/div.accordion-item-->
                </div><!--/div.accordion-flush-->
              </div><!--div.sidenav-view-selector -->
            </div><!--/div.col -->

            <hr></div><!--/div.mobile-row -->

          <div class="accordion accordion-flush" id="accordionFlush11">
            <div class="accordion-item">

              <button id="chapters" class="accordion-button show" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapseEleven" aria-expanded="false" aria-controls="flush-collapseEleven">
                <h2 class="accordion-header chapters" id="flush-headingEleven">
                  EPISODES
                </h2>
              </button>
              <div id="flush-collapseEleven" class="accordion-collapse show collapse" aria-labelledby="flush-headingEleven" data-bs-parent="#accordionFlush11">

                <div class="accordion-body">
                  <div class="accordion accordion-flush" id="accordionFlush1">
  <div class="accordion-item">
    <div class="accordion-header" id="flush-heading1">
        <a href="index.html">Marine Biological Data Mobilization Workshop 2024</a>
    </div><!--/div.accordion-header-->

  </div><!--/div.accordion-item-->
</div><!--/div.accordion-flush-->

<div class="accordion accordion-flush" id="accordionFlush2">
  <div class="accordion-item">
    <div class="accordion-header" id="flush-heading2">
        <a href="01-introduction.html">1. Introduction to Darwin Core</a>
    </div><!--/div.accordion-header-->

  </div><!--/div.accordion-item-->
</div><!--/div.accordion-flush-->

<div class="accordion accordion-flush" id="accordionFlush3">
  <div class="accordion-item">
    <div class="accordion-header" id="flush-heading3">
        <a href="02-break.html">2. Social Break</a>
    </div><!--/div.accordion-header-->

  </div><!--/div.accordion-item-->
</div><!--/div.accordion-flush-->

<div class="accordion accordion-flush" id="accordionFlush4">
  <div class="accordion-item">
    <div class="accordion-header" id="flush-heading4">
        <a href="03-data-cleaning.html">3. Data Cleaning</a>
    </div><!--/div.accordion-header-->

  </div><!--/div.accordion-item-->
</div><!--/div.accordion-flush-->

<div class="accordion accordion-flush" id="accordionFlush5">
  <div class="accordion-item">
    <div class="accordion-header" id="flush-heading5">
        <a href="04-create-schema.html">4. Darwin Core and Extension Schemas</a>
    </div><!--/div.accordion-header-->

  </div><!--/div.accordion-item-->
</div><!--/div.accordion-flush-->

<div class="accordion accordion-flush" id="accordionFlush6">
  <div class="accordion-item">
    <div class="accordion-header" id="flush-heading6">
        <a href="05-break.html">5. Social Break</a>
    </div><!--/div.accordion-header-->

  </div><!--/div.accordion-item-->
</div><!--/div.accordion-flush-->

<div class="accordion accordion-flush" id="accordionFlush7">
  <div class="accordion-item">
    <div class="accordion-header" id="flush-heading7">
        <a href="06-qa-qc.html">6. QA/QC</a>
    </div><!--/div.accordion-header-->

  </div><!--/div.accordion-item-->
</div><!--/div.accordion-flush-->

<div class="accordion accordion-flush" id="accordionFlush8">
  <div class="accordion-item">
    <div class="accordion-header" id="flush-heading8">
        <a href="07-validation-and-publishing.html">7. Metadata and publishing</a>
    </div><!--/div.accordion-header-->

  </div><!--/div.accordion-item-->
</div><!--/div.accordion-flush-->

<div class="accordion accordion-flush" id="accordionFlush9">
  <div class="accordion-item">
    <div class="accordion-header" id="flush-heading9">
        <a href="08-post-workshop.html">8. Continuing the Conversation</a>
    </div><!--/div.accordion-header-->

  </div><!--/div.accordion-item-->
</div><!--/div.accordion-flush-->

                </div>
              </div>
            </div>

            <hr class="half-width"><div class="accordion accordion-flush lesson-resources" id="accordionFlush12">
              <div class="accordion-item">
                <h2 class="accordion-header" id="flush-headingTwelve">
                  <button class="accordion-button collapsed" id="lesson-resources" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapseTwelve" aria-expanded="false" aria-controls="flush-collapseTwelve">
                    RESOURCES
                  </button>
                </h2>
                <div id="flush-collapseTwelve" class="accordion-collapse collapse" aria-labelledby="flush-headingTwelve" data-bs-parent="#accordionFlush12">
                  <div class="accordion-body">
                    <ul><li>
                        <a href="../instructor/key-points.html">Key Points</a>
                      </li>
                      <li>
                        <a href="../instructor/instructor-notes.html">Instructor Notes</a>
                      </li>
                      <li>
                        <a href="../instructor/images.html">Extract All Images</a>
                      </li>
                      <hr><li><a class="dropdown-item" href="reference.html">Reference</a></li><li><a class="dropdown-item" href="about.html">About</a></li><li><a class="dropdown-item" href="helpful-tutorials.html">Helpful Tutorials</a></li><li><a class="dropdown-item" href="edna-extension.html">DNA-derived Data Extension</a></li><li><a class="dropdown-item" href="acoustic-telemetry.html">Acoustic (and Satellite) Telemetry</a></li>
                    </ul></div>
                </div>
              </div>
            </div>
            <hr class="half-width lesson-resources"><a href="../instructor/aio.html">See all in one page</a>


            <hr class="d-none d-sm-block d-md-none"><div class="d-grid gap-1">

            </div>
          </div><!-- /div.accordion -->
        </div><!-- /div.sidebar-inner -->
      </nav></div><!-- /div.sidebar -->
  </div><!-- /div.sidebar-col -->
<!-- END:   inst/pkgdown/templates/navbar.html-->

        <!-- START: inst/pkgdown/templates/content-instructor.html -->
  <div class="col-xl-8 col-lg-12 primary-content">
    <nav class="lesson-content mx-md-4" aria-label="Previous and Next Chapter"><!-- content for small screens --><div class="d-block d-sm-block d-md-none">
        <a class="chapter-link" href="../instructor/index.html"><i aria-hidden="true" class="small-arrow" data-feather="arrow-left"></i>Previous</a>
        <a class="chapter-link float-end" href="../instructor/index.html">Next<i aria-hidden="true" class="small-arrow" data-feather="arrow-right"></i></a>
      </div>
      <!-- content for large screens -->
      <div class="d-none d-sm-none d-md-block">
        <a class="chapter-link" href="../instructor/index.html" rel="prev">
          <i aria-hidden="true" class="small-arrow" data-feather="arrow-left"></i>
          Previous: Marine Biological
        </a>
        <a class="chapter-link float-end" href="../instructor/index.html" rel="next">
          Next: Marine Biological...
          <i aria-hidden="true" class="small-arrow" data-feather="arrow-right"></i>
        </a>
      </div>
      <hr></nav><main id="main-content" class="main-content"><div class="container lesson-content">
        <h1>DNA-derived Data Extension</h1>
        <p>Last updated on 2025-02-28 |

        <a href="https://github.com/MathewBiddle/test_workshop/edit/main/learners/edna-extension.md" class="external-link">Edit this page <i aria-hidden="true" data-feather="edit"></i></a></p>





        <div class="text-end">
          <button role="button" aria-pressed="false" tabindex="0" id="expand-code" class="pull-right" data-expand="Expand All Solutions " data-collapse="Collapse All Solutions "> Expand All Solutions <i aria-hidden="true" data-feather="plus"></i></button>
        </div>

        

<div class="overview card">
<h2 class="card-header">Overview</h2>
<div class="row g-0">
<div class="col-md-4">
<div class="card-body">
<div class="inner">
<h3 class="card-title">Questions</h3>
<ul><li>How should people use the DNA-derived data extension?</li>
<li>Why should people the DNA-derived data extension?</li>
<li>What are the required or highly recommended Darwin Core terms for
sharing to OBIS?</li>
</ul></div>
</div>
</div>
<div class="col-md-8">
<div class="card-body">
<div class="inner bordered">
<h3 class="card-title">Objectives</h3>
<ul><li>Understand the purpose of the DNA-derived data extension.</li>
<li>Understand required or (highly) recommended fields needed in the
extension, depending on analytical approach taken.</li>
</ul></div>
</div>
</div>
</div>
</div>
<figure><img align="center" src="../bio_mobilization_workshop/assets/img/DwC-A_closeup.png" alt="DNA-derived extension in Darwin Core" class="figure"><figcaption class="figure-caption text-center">
Source: GBIF DNA-derived Data Extension Manual
</figcaption></figure><section><h2 class="section-heading" id="introduction">Introduction<a class="anchor" aria-label="anchor" href="#introduction"></a></h2>
<hr class="half-width"><p>DNA-derived biological occurrence data allow us to observe
inconspicuous and cryptic taxa. DNA-derived data may come from sampling
the environment (e.g. water, sediment), bulk organisms, or individual
organisms. The derived data may be backed by preserved physical material
or not, and may result from genetic sequencing or other DNA detection
methods, such as qPCR.</p>
<p>Like all biological observation data, DNA-derived data should be as
standardized and reproducible as possible, regardless of whether or not
the detected species have formal scientific names. It is recommended
practice that processed data should be provided as an extension to the
occurrence core/extension in the DwC-A schema. The number of DNA-derived
datasets in OBIS is limited (currently only 26) and our goal is to
mobilize more datasets through this workshop. In this workshop we will
focus on the two analytical methods that are typically used in studying
environmental DNA (eDNA):</p>
<ol style="list-style-type: decimal"><li>Those which aim to detect a specific organism (qPCR/ddPCR)</li>
<li>Those which describe an assemblage or community of organisms
(metabarcoding).</li>
</ol><div class="section level3">
<h3 id="qpcrddpcr">qPCR/ddPCR<a class="anchor" aria-label="anchor" href="#qpcrddpcr"></a></h3>
<p>For the detection of specific species in eDNA-samples, most analyses
include species-specific primers and qPCR (Quantitative Polymerase Chain
Reaction) or ddPCR (Droplet-Digital Polymerase Chain Reaction). These
methods do not generate DNA-sequences, and the occurrence data are
completely dependent on the specificity of primers or assays used, so it
is highly recommended that this information be preserved and shared with
the data.</p>
</div>
<div class="section level3">
<h3 id="metabarcoding">Metabarcoding<a class="anchor" aria-label="anchor" href="#metabarcoding"></a></h3>
<p>Metabarcoding utilizes general primers to recover hundreds to
millions of DNA-sequences from a DNA extract. Sequences are typically
identified and grouped, either by clustering sequences into operational
taxonomic units (OTUs) or by denoising the data, which produces amplicon
sequence variants (ASVs). The sequences that have been identified can be
assigned a taxon by comparing them against reference databases,
sometimes called reference libraries. To avoid confusion with the
“libraries” that are the results of the sequencing process, we will try
to refer to them as reference databases.</p>
<div class="section level4">
<h4 id="archiving-raw-sequences">Archiving Raw Sequences<a class="anchor" aria-label="anchor" href="#archiving-raw-sequences"></a></h4>
<p>The raw, pre-processed sequences should be preserved in external
archives or repositories (e.g. <a href="https://www.ncbi.nlm.nih.gov/genbank/" class="external-link">GenBank</a>, <a href="https://www.ebi.ac.uk/ena/browser/" class="external-link">ENA</a>).</p>
</div>
<div class="section level4">
<h4 id="taxonomic-assignment">Taxonomic Assignment<a class="anchor" aria-label="anchor" href="#taxonomic-assignment"></a></h4>
<p>A representative sequence for each OTU/ASV can be assigned a taxon by
comparing them against reference databases. The efficacy of assignment
depends on the completeness (coverage) ande reliability of reference
libraries, as well as the tools used to carry out the classification.
For example, many assignment algorithms are based on some kind of
minimum confidence threshold and may result in OTUs/ASVs that are
unassigned or badly assigned. These are all moving targets, making it
essential to apply taxonomic expertise and caution in the interpretation
of results.</p>
<p>It may be that you are only able to confidently assign your sequences
a high-level classification, like Kingdom. However, it is still worth
sharing standardized data and metadata so they can be re-analyzed and
built upon as methods and reference databases improve. For genetic
sequencing data to be reusable and reproducible, it is important to
provide data users with information about the provanance and processing
of the DNA. For example, habitat where the sample was collected from,
DNA sequence, primers used, targeted gene, and referencing the sampling
protocol . When applicable, providing the verified amplicon sequence
variants (<a href="https://doi.org/10.1038/ismej.2017.119" class="external-link">Сallahan et
al. 2017</a>) allow for precise reinterpretation of data, intra-specific
population genetic analyses (<a href="https://doi.org/10.1038/s41559-016-0004" class="external-link">Sigsgaard et
al. 2019</a>) and is likely to increase identification accuracy.</p>
</div>
</div>
</section><section><h2 class="section-heading" id="workflow">Workflow<a class="anchor" aria-label="anchor" href="#workflow"></a></h2>
<hr class="half-width"><p>The following workflow diagram is taken from the GBIF DNA-derived
data manuscript. We are mainly focusing on the lower half of this
diagram, particularly in standardizing genomic data to DwC and providing
proper referencing to protocols and other metadata required for
understanding the context in which the data was collected.</p>
<figure><img src="https://docs.gbif.org/publishing-dna-derived-data/1.0/img/web/outline-of-a-platform.en.svg" alt="DNA-derived data workflow" class="figure"><figcaption>
Source: GBIF DNA-derived Data Extension Manual.
</figcaption></figure><div class="section level3">
<h3 id="dna-derived-extension">DNA-derived extension<a class="anchor" aria-label="anchor" href="#dna-derived-extension"></a></h3>
<p>The DNA-derived data extension is an extension in the DwC-A schema,
and has to be used as an extension to Occurrence Core. It can be used in
combination with other extensions, such as the Measurement &amp; Facts
extension. Information included in the DNA-derived extension should make
it easier for data users to understand, analyze or reuse your genomic
occurrence data. The value in adding your DNA-sequence or genomic data
to GBIF comes from associating spatio-temporal occurrence data and
DNA-based names from the data. It helps provide a mechanism for storing
occurrence records of undescribed species. When these species become
linked to a Linnean name, all these linked occurrence records will be
immediately available. Including this data in GBIF increases citability,
can hasten its discovery and integration in biological conservation and
policy-making. Specific granularity is required for accurate
reproducibility, especially in a field where protocols used can vastly
impact the taxa observed.</p>
</div>
<div class="section level3">
<h3 id="categories-of-dna-derived-data">5 Categories of DNA-derived data<a class="anchor" aria-label="anchor" href="#categories-of-dna-derived-data"></a></h3>
<p>Typically, molecular approaches to biodiversity characterization
through qPCR and metabarcoding can be separated into 5 categories.
Information collected in each of these categories can be included in the
DNA-derived extension.</p>
<ul><li>Category 1 (metabarcoding): DNA-derived occurrences = DNA sequence
or detection through PCR is the only evidence for presence of a given
organism or community</li>
<li>Category 2 (metabarcoding): Enriched occurrences = if some genetic
materials are, or can be, associated with an observation or a specimen
(not the only evidence of occurrences).</li>
<li>Category 3 (qPCR): Targeted species detection (qPCR/ddPCR) = data
where a specific (qPCR/ddPCR) assay is used to detect presence/absence
of DNA sequence specific to target organism.</li>
<li>Category 4 (metabarcoding): Name references (e.g. iBOL) =
DNA-derived names, derived from clustering (OTU) or denoising (ASV)</li>
<li>Category 5 (metabarcoding): Metadata-only datasets</li>
</ul><p>See for a decision tree <a href="https://docs.gbif.org/publishing-dna-derived-data/1.0/en/" class="external-link">Table
1</a>. For examples of datasets in GBIF for each of these categories see
sections 2.1.1. - 2.1.5.</p>
</div>
<div class="section level3">
<h3 id="tables">Tables<a class="anchor" aria-label="anchor" href="#tables"></a></h3>
<p>The DNA-derived data extension will have different requirements based
on the analytical method applied in your project (i.e. metabarcoding
vs. qPCR). Although few fields of the DNA-derived extension are
“required” for metabarcoding and qPCR data, the decision to exclude
“highly recommended” fields could severely impact how reusable your data
will be to others. Therefore, when dealing with metabarcoding or qPCR
data, it is (highly) recommended that aside from the general required
fields in DwC the following (free-text) fields are also included:</p>
<p><strong>Table 1.</strong> Recommended fields for Occurrence core or
extension. An extended version of this table is found in <a href="https://docs.gbif-uat.org/publishing-dna-derived-data/1.0/en/#mapping-metabarcoding-edna-and-barcoding-data" class="external-link">GBIF
manual</a>.</p>
<table class="table"><colgroup><col width="32%"><col width="25%"><col width="21%"><col width="21%"></colgroup><thead><tr class="header"><th align="left"></th>
<th align="left">Metabarcoding</th>
<th align="left">qPCR</th>
<th align="left"></th>
</tr></thead><tbody><tr class="odd"><td align="left"><strong>Darwin Core Term</strong></td>
<td align="left"><strong>Description</strong></td>
<td align="left"><strong>Description</strong></td>
<td align="left"><strong>Required</strong></td>
</tr><tr class="even"><td align="left"><a href="https://dwc.tdwg.org/terms/#dwc:organismQuantity" class="external-link"><code>organismQuantity</code></a></td>
<td align="left">Number of reads of this sequence variant in the
sample</td>
<td align="left">Number of positive droplets/chambers in the sample</td>
<td align="left">Highly recommended</td>
</tr><tr class="odd"><td align="left"><a href="https://dwc.tdwg.org/terms/#dwc:organismQuantityType" class="external-link"><code>organismQuantityType</code></a></td>
<td align="left">Should always be <strong>DNA sequence
reads</strong>
</td>
<td align="left">The partition type</td>
<td align="left">Highly recommended</td>
</tr><tr class="even"><td align="left"><a href="https://dwc.tdwg.org/terms/#dwc:sampleSizeValue" class="external-link"><code>sampleSizeValue</code></a></td>
<td align="left">Total number of reads in the sample. This important
since it allows calculating the relative abundance of the sequence
variant within the sample</td>
<td align="left">The number of accepted partitions (n), e.g. meaning
accepted droplets in ddPCR or chambers in dPCR.</td>
<td align="left">Highly recommended</td>
</tr><tr class="odd"><td align="left"><a href="https://dwc.tdwg.org/terms/#dwc:sampleSizeUnit" class="external-link"><code>sampleSizeUnit</code></a></td>
<td align="left">Should always be <strong>DNA sequence
reads</strong>
</td>
<td align="left">The partition type, should be equal to the value in
organismQuantityType</td>
<td align="left">Highly recommended</td>
</tr><tr class="even"><td align="left"><a href="https://dwc.tdwg.org/terms/#dwc:materialSampleID" class="external-link"><code>materialSampleID</code></a></td>
<td align="left">An identifier for the MaterialSample (as opposed to a
particular digital record of the material sample). Use the biosample ID
if one was obtained from a nucleotide archive. In the absence of a
persistent global unique identifier, construct one from a combination of
identifiers in the record that will most closely make the
materialSampleID globally unique.</td>
<td align="left">An identifier for the MaterialSample (as opposed to a
particular digital record of the material sample). Use the biosample ID
if one was obtained from a nucleotide archive. In the absence of a
persistent global unique identifier, construct one from a combination of
identifiers in the record that will most closely make the
materialSampleID globally unique.</td>
<td align="left">Highly recommended</td>
</tr><tr class="odd"><td align="left"><a href="https://dwc.tdwg.org/terms/#dwc:samplingProtocol" class="external-link"><code>samplingProtocol</code></a></td>
<td align="left">The name of, or reference to, or description of the
method or protocol used during the sampling event.</td>
<td align="left">The name of, or reference to, or description of the
method or protocol used during the sampling event.</td>
<td align="left">Recommended</td>
</tr><tr class="even"><td align="left"><a href="https://dwc.tdwg.org/terms/#dwc:taxonID" class="external-link"><code>taxonID</code></a></td>
<td align="left">For eDNA data, it is recommended to use an MD5 hash of
the sequence and prepend it with <strong>ASV</strong>.</td>
<td align="left"></td>
<td align="left">metabarcoding: Highly recommended if DNA_sequence is
not provided. for qPCR data, this field is not required.</td>
</tr><tr class="odd"><td align="left"><a href="https://dwc.tdwg.org/terms/#dwc:scientificName" class="external-link"><code>scientificName</code></a></td>
<td align="left"></td>
<td align="left">Latin name of the closest known taxon (species or
higher) or an OTU identifier from BOLD or UNITE</td>
<td align="left">metabarcoding: not required. qPCR:
<strong>Required</strong>
</td>
</tr></tbody></table><p>After populating the Occurrence Core or extension with the
(additional) required or highly recommended fields, additional
information related to either your metabarcoding or qPCR data is to be
included in the DNA-derived data extension. The (highly) recommended
fields to include in this extension depend on whether your output is the
result of metabarcode or qPCR analysis. The tables below only indicate
the highly recommended or required fields - additional field should be
included if applicable (see <a href="https://docs.gbif.org/publishing-dna-derived-data/1.0/en/" class="external-link">GBIF
DNA-derived Data Extension Manual</a>).</p>
<div id="accordionSpoiler1" class="accordion spoiler-accordion accordion-flush">
<div class="accordion-item">
<button class="accordion-button spoiler-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#collapseSpoiler1" aria-expanded="false" aria-controls="collapseSpoiler1">
  <h3 class="accordion-header" id="headingSpoiler1">
  <div class="note-square"><i aria-hidden="true" class="callout-icon" data-feather="eye"></i></div>
  DNA-derived Data Extension Terms for Metabarcoding Data
  </h3>
</button>
<div id="collapseSpoiler1" class="accordion-collapse collapse" data-bs-parent="#accordionSpoiler1" aria-labelledby="headingSpoiler1">
<div class="accordion-body">
<table class="table"><colgroup><col width="19%"><col width="38%"><col width="41%"></colgroup><thead><tr class="header"><th>Darwin Core Term</th>
<th>Description</th>
<th>Required</th>
</tr></thead><tbody><tr class="odd"><td><a href="https://rs.gbif.org/extension/gbif/1.0/dna_derived_data_2021-07-05.xml#DNA_sequence" class="external-link"><code>DNA_sequence</code></a></td>
<td>The DNA sequence (ASV).</td>
<td>Highly recommended</td>
</tr><tr class="even"><td><a href="https://rs.gbif.org/extension/gbif/1.0/dna_derived_data_2021-07-05.xml#sop" class="external-link"><code>sop</code></a></td>
<td>Standard operating procedures used in assembly and/or annotation of
genomes, metagenomes or environmental sequences (e.g. through
protocols.io)</td>
<td>Recommended</td>
</tr><tr class="odd"><td><a href="https://rs.gbif.org/extension/gbif/1.0/dna_derived_data_2021-07-05.xml#target_gene" class="external-link"><code>target_gene</code></a></td>
<td>Targeted gene or marker name for marker-based studies</td>
<td>Highly recommended</td>
</tr><tr class="even"><td><a href="https://rs.gbif.org/extension/gbif/1.0/dna_derived_data_2021-07-05.xml#target_gene" class="external-link"><code>target_subfragment</code></a></td>
<td>Name of subfragment of a gene or markerImportant to e.g. identify
special regions on marker genes like the hypervariable V6 region of the
16S rRNA gene</td>
<td>Highly recommended</td>
</tr><tr class="odd"><td><a href="https://rs.gbif.org/extension/gbif/1.0/dna_derived_data_2021-07-05.xml#pcr_primer_forward" class="external-link"><code>pcr_primer_forward</code></a></td>
<td>Forward PCR primer that was used to amplify the sequence of the
targeted gene, locus or subfragment.</td>
<td>Highly recommended</td>
</tr><tr class="even"><td><a href="https://rs.gbif.org/extension/gbif/1.0/dna_derived_data_2021-07-05.xml#pcr_primer_reverse" class="external-link"><code>pcr_primer_reverse</code></a></td>
<td>Reverse PCR primer that was used to amplify the sequence of the
targeted gene, locus or subfragment.</td>
<td>Highly recommended</td>
</tr><tr class="odd"><td><a href="https://rs.gbif.org/extension/gbif/1.0/dna_derived_data_2021-07-05.xml#pcr_primer_name_forward" class="external-link"><code>pcr_primer_name_forward</code></a></td>
<td>Name of the forward PCR primer</td>
<td>Highly recommended</td>
</tr><tr class="even"><td><a href="https://rs.gbif.org/extension/gbif/1.0/dna_derived_data_2021-07-05.xml#pcr_primer_name_reverse" class="external-link"><code>pcr_primer_name_reverse</code></a></td>
<td>Name of the reverse PCR primer</td>
<td>Highly recommended</td>
</tr><tr class="odd"><td><a href="https://rs.gbif.org/extension/gbif/1.0/dna_derived_data_2021-07-05.xml#pcr_primer_reference" class="external-link"><code>pcr_primer_reference</code></a></td>
<td>Reference for the primers</td>
<td>Highly recommended</td>
</tr></tbody></table></div>
</div>
</div>
</div>
<div id="accordionSpoiler2" class="accordion spoiler-accordion accordion-flush">
<div class="accordion-item">
<button class="accordion-button spoiler-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#collapseSpoiler2" aria-expanded="false" aria-controls="collapseSpoiler2">
  <h3 class="accordion-header" id="headingSpoiler2">
  <div class="note-square"><i aria-hidden="true" class="callout-icon" data-feather="eye"></i></div>
  DNA-derived Data Extension Terms for qPCR Data
  </h3>
</button>
<div id="collapseSpoiler2" class="accordion-collapse collapse" data-bs-parent="#accordionSpoiler2" aria-labelledby="headingSpoiler2">
<div class="accordion-body">
<table class="table"><colgroup><col width="19%"><col width="38%"><col width="41%"></colgroup><thead><tr class="header"><th>Darwin Core Term</th>
<th>Description</th>
<th>Required</th>
</tr></thead><tbody><tr class="odd"><td><a href="https://rs.gbif.org/extension/gbif/1.0/dna_derived_data_2021-07-05.xml#sop" class="external-link"><code>sop</code></a></td>
<td>Standard operating procedures used in assembly and/or annotation of
genomes, metagenomes or environmental sequences. A reference to a well
documented protocol, e.g. using protocols.io</td>
<td>Highly recommended</td>
</tr><tr class="even"><td><a href="https://rs.gbif.org/extension/gbif/1.0/dna_derived_data_2021-07-05.xml#annealingTemp" class="external-link"><code>annealingTemp</code></a></td>
<td>The reaction temperature during the annealing phase of PCR.</td>
<td>Required if annealingTemp was supplied</td>
</tr><tr class="odd"><td><a href="https://rs.gbif.org/extension/gbif/1.0/dna_derived_data_2021-07-05.xml#annealingTempUnit" class="external-link"><code>annealinTempUnit</code></a></td>
<td></td>
<td>Highly recommended</td>
</tr><tr class="even"><td><a href="https://rs.gbif.org/extension/gbif/1.0/dna_derived_data_2021-07-05.xml#pcr_cond" class="external-link"><code>pcr_cond</code></a></td>
<td>Description of reaction conditions and components of PCR in the form
of “initial denaturation:94degC_1.5min; annealing=…”</td>
<td>Highly recommended</td>
</tr><tr class="odd"><td><a href="https://rs.gbif.org/extension/gbif/1.0/dna_derived_data_2021-07-05.xml#probeReporter" class="external-link"><code>probeReporter</code></a></td>
<td>Type of fluorophore (reporter) used. Probe anneals within amplified
target DNA. Polymerase activity degrades the probe that has annealed to
the template, and the probe releases the fluorophore from it and breaks
the proximity to the quencher, thus allowing fluorescence of the
fluorophore.</td>
<td>Highly recommended</td>
</tr><tr class="even"><td><a href="https://rs.gbif.org/extension/gbif/1.0/dna_derived_data_2021-07-05.xml#probeQuencher" class="external-link"><code>probeQuencher</code></a></td>
<td>Type of quencher used. The quencher molecule quenches the
fluorescence emitted by the fluorophore when excited by the cycler’s
light source as long as fluorophore and the quencher are in proximity,
quenching inhibits any fluorescence signals.</td>
<td>Highly recommended</td>
</tr><tr class="odd"><td><a href="https://rs.gbif.org/extension/gbif/1.0/dna_derived_data_2021-07-05.xml#ampliconSize" class="external-link"><code>ampliconSize</code></a></td>
<td>The length of the amplicon in basepairs</td>
<td>Highly recommended</td>
</tr><tr class="even"><td><a href="https://rs.gbif.org/extension/gbif/1.0/dna_derived_data_2021-07-05.xml#thresholdQuantificationCycle" class="external-link"><code>thresholdQuantificationCycle</code></a></td>
<td>Threshold for change in fluorescence signal between cycles</td>
<td>qPCR: Highly recommended</td>
</tr><tr class="odd"><td><a href="https://rs.gbif.org/extension/gbif/1.0/dna_derived_data_2021-07-05.xml#baselineValue" class="external-link"><code>baselineValue</code></a></td>
<td>The number of cycles when fluorescence signal from the target
amplification is below background fluorescence not originated from the
real target amplification.</td>
<td>qPCR: Highly recommended</td>
</tr></tbody></table></div>
</div>
</div>
</div>
<p>It is recommended that if ASVs are provided, MD5s should be generated
by the biodiversity discovery platforms. If ASVs are not provided, the
MD5s need to be mandatory. This will help data platform index actual
sequences, or at the very minimum a MD5 checksum of these to facilitate
searches for ASVs across datasets.</p>
</div>
</section><section><h2 class="section-heading" id="references">References<a class="anchor" aria-label="anchor" href="#references"></a></h2>
<hr class="half-width"><ol style="list-style-type: decimal"><li>Andersson AF, Bissett A, Finstad AG, Fossøy F, Grosjean M, Hope M,
Jeppesen TS, Kõljalg U, Lundin D, Nilsson RN, Prager M, Svenningsen C
&amp; Schigel D (2021) Publishing DNA-derived data through biodiversity
data platforms. v1.0 Copenhagen: GBIF Secretariat. <a href="https://doi.org/10.35035/doc-vf1a-nr22" class="external-link">https://doi.org/10.35035/doc-vf1a-nr22</a>.</li>
<li><a href="https://manual.obis.org/examples.html" class="external-link">The OBIS Manual:
Examples</a></li>
<li><a href="http://rs.gbif.org/extension/gbif/1.0/dna_derived_data_2021-07-05.xml" class="external-link">Schema
for DNA-Derived Data Extension</a></li>
<li><a href="https://www.youtube.com/watch?v=KKh_Hd8zybs" class="external-link">OBIS Webinar
on Genetic Data 28 Oct 2021</a></li>
</ol></section><section><h2 class="section-heading" id="example-datasets">Example Datasets<a class="anchor" aria-label="anchor" href="#example-datasets"></a></h2>
<hr class="half-width"><ol style="list-style-type: decimal"><li><a href="https://obis.org/dataset/62b97724-da17-4ca7-9b26-b2a22aeaab51" class="external-link">18S
Monterey Bay Time Series: an eDNA data set from Monterey Bay,
California, including years 2006, 2013 - 2016</a></li>
<li><a href="https://obis.org/dataset/4fd52bef-1d35-4e06-8281-7213b6dde690" class="external-link">Passive
eDNA collection enhances aquatic biodiversity analysis, Australia
(2019)</a></li>
</ol></section><section><h2 class="section-heading" id="relevant-rpython-packages">Relevant R/Python packages<a class="anchor" aria-label="anchor" href="#relevant-rpython-packages"></a></h2>
<hr class="half-width"><p>R: - <a href="https://benjjneb.github.io/dada2/" class="external-link">DADA2: processing
sequences to ASVs and more</a> - <a href="https://joey711.github.io/phyloseq/" class="external-link">Phyloseq: tool to import,
store, analyze, and graphically display complex phylogenetic sequencing
data after clustering</a></p>
<p>Python: - <a href="https://github.com/biocore/biom-format" class="external-link uri">https://github.com/biocore/biom-format</a> - <a href="https://entrezpy.readthedocs.io/en/master/" class="external-link uri">https://entrezpy.readthedocs.io/en/master/</a></p>
<div id="keypoints1" class="callout keypoints">
<div class="callout-square">
<i class="callout-icon" data-feather="key"></i>
</div>
<div class="callout-inner">
<h3 class="callout-title">Key Points</h3>
<div class="callout-content">
<ul><li>“Ensure you know the objective of your study.”</li>
<li>“Define whether you have used metabarcoding or qPCR to determine
(highly) recommended fields in the DNA-derived data extension.”</li>
</ul></div>
</div>
</div>
<!--
Place links that you need to refer to multiple times across pages here. Delete
any links that you are not going to use.
 -->
</section></div> <!-- / div.lesson-content -->
    </main><!-- / main#main-content.main-content --><nav class="bottom-pagination mx-md-4" aria-label="Previous and Next Chapter"><div class="d-block d-sm-block d-md-none">
        <a class="chapter-link" href="../instructor/index.html"><i aria-hidden="true" class="small-arrow" data-feather="arrow-left"></i>Previous</a>
        <a class="chapter-link float-end" href="../instructor/index.html">Next<i aria-hidden="true" class="small-arrow" data-feather="arrow-right"></i></a>
      </div>
      <!-- content for large screens -->
      <div class="d-none d-sm-none d-md-block">
        <a class="chapter-link" href="../instructor/index.html" rel="prev">
          <i aria-hidden="true" class="small-arrow" data-feather="arrow-left"></i>
          Previous: Marine Biological
        </a>
        <a class="chapter-link float-end" href="../instructor/index.html" rel="next">
          Next: Marine Biological...
          <i aria-hidden="true" class="small-arrow" data-feather="arrow-right"></i>
        </a>
      </div>
    </nav></div> <!-- / div.primary-content.col-xs-12 -->
<!-- END:   inst/pkgdown/templates/content-instructor.html-->

      </div><!--/div.row-->
      		<footer class="row footer mx-md-3"><hr><div class="col-md-6">
        <p>This lesson is subject to the <a href="CODE_OF_CONDUCT.html">Code of Conduct</a></p>
        <p>

        <a href="https://github.com/MathewBiddle/test_workshop/edit/main/learners/edna-extension.md" class="external-link">Edit on GitHub</a>

	
        | <a href="https://github.com/MathewBiddle/test_workshop/blob/main/CONTRIBUTING.md" class="external-link">Contributing</a>
        | <a href="https://github.com/MathewBiddle/test_workshop/" class="external-link">Source</a></p>
				<p><a href="https://github.com/MathewBiddle/test_workshop/blob/main/CITATION.cff" class="external-link">Cite</a> | <a href="mailto:https://obis.org/contact/" class="external-link">Contact</a> | <a href="https://carpentries.org/about/" class="external-link">About</a></p>
			</div>
			<div class="col-md-6">

        <p>Materials licensed under <a href="LICENSE.html">CC-BY 4.0</a> by the authors</p>

        <p>Template licensed under <a href="https://creativecommons.org/licenses/by-sa/4.0/" class="external-link">CC-BY 4.0</a> by <a href="https://carpentries.org/" class="external-link">The Carpentries</a></p>
        <p>Built with <a href="https://github.com/carpentries/sandpaper/tree/0.16.11" class="external-link">sandpaper (0.16.11)</a>, <a href="https://github.com/carpentries/pegboard/tree/0.7.9" class="external-link">pegboard (0.7.9)</a>, and <a href="https://github.com/carpentries/varnish/tree/1.0.5" class="external-link">varnish (1.0.5)</a></p>
			</div>
		</footer></div> <!-- / div.container -->
	<div id="to-top">
		<a href="#top">
      <i class="search-icon" data-feather="arrow-up" role="img" aria-label="Back To Top"></i><br><!-- <span class="d-none d-sm-none d-md-none d-lg-none d-xl-block">Back</span> To Top --><span class="d-none d-sm-none d-md-none d-lg-none d-xl-block">Back</span> To Top
		</a>
	</div>
  <script type="application/ld+json">
    {
  "@context": "https://schema.org",
  "@type": "TrainingMaterial",
  "@id": "https://MathewBiddle.github.io/test_workshop/instructor/edna-extension.html",
  "inLanguage": "en",
  "dct:conformsTo": "https://bioschemas.org/profiles/TrainingMaterial/1.0-RELEASE",
  "description": "A Carpentries Lesson teaching foundational data and coding skills to researchers worldwide",
  "keywords": "software, data, lesson, The Carpentries",
  "name": "DNA-derived Data Extension",
  "creativeWorkStatus": "active",
  "url": "https://MathewBiddle.github.io/test_workshop/instructor/edna-extension.html",
  "identifier": "https://MathewBiddle.github.io/test_workshop/instructor/edna-extension.html",
  "dateCreated": "2024-04-19",
  "dateModified": "2025-02-28",
  "datePublished": "2025-04-29"
}

  </script><script>
		feather.replace();
	</script></body></html><!-- END:   inst/pkgdown/templates/layout.html-->

